content/developer-guide/toc.yml

```diff
@@ -1,5 +1,3 @@
-- name: Tenant management
-  href: tenant-dev/dev-guide-tenant-management.md
```

Comment:
Why did you remove this topic from the TOC? Did you deem it content that isn't worthwhile (which I would agree with--I'm more curious why you removed it).

Reply:
It's basically a duplicate of what's in the API guide, without adding anything.